config.go

```diff
@@ -36,6 +41,9 @@ type Config struct {
 	// Simnet is set to true when using bitcoind's regtest.
 	Regtest bool `long:"regtest" description:"Use regtest"`
 
+	// ChainConn specifies whether to attempt connecting to a bitcoin backend.
+	ChainConn bool `long:"chainconn" description:"Whether to attempt to connect to a backing bitcoin. Some endpoints will not be available if this option is not enabled."`
```

Comment:
nit: s/backing bitcoin/backing bitcoin node/

Reply:
Also, what do you think about `--connect-bitcoin` as an alternative name for the flag?